guides/plugins/apps/content/cms/add-custom-cms-blocks.md

```diff
@@ -20,36 +20,38 @@ The basic directory structure looks as follows:
 
 ```text
 ├── Resources
+│   ├── app
+│   │   └── storefront
+│   │       └── src
+│   │           └── scss
+│   │               └── base.scss
 │   ├── cms
 │   │   └── blocks
-│   │       ├── my-first-block
+│   │       ├── swag-image-text-reversed
 │   │       │   ├── preview.html
 │   │       │   └── styles.css
-│   │       └── my-second-block
-│   │           ├── preview.html
-│   │           └── styles.css
 │   ├── views
 │   │   └── storefront
-│   │       └── cms
-│   │           └── block
-│   │               ├── my-first-block.html.twig
-│   │               └── my-second-block.html.twig
+│   │       └── block
+│   │           └── cms-block-swag-image-text-reversed-component.html.twig
 │   └── cms.xml
 └── manifest.xml
 ```
 
 Each CMS block defined within your `cms.xml` must have a directory matching the block's name in `Resources/cms/blocks/`.
-In those directories you shape your blocks by supplying a `preview.html` containing the template used for displaying a preview.
-Styling the preview is possible from the `styles.css`.
+In those directories you shape your blocks for the CMS editor in the Administration by supplying a `preview.html` containing the template used for displaying a preview.
+Styling the preview in the sidebar and the component on the CMS stage is possible from the `styles.css`.
 
 {% hint type=info %}
 Due to technical limitations it's not possible to use templating engines \(like Twig\) or preprocessors \(like Sass\) for rendering and styling the preview.
 {% endhint %}
 
+The Storefront representations reside in the `Resources/views/storefront/block/` directory.
```

Comment:
```suggestion
The Storefront representations of our new block(s) reside in the `Resources/views/storefront/block/` directory.
```

Reply:
To stay consistent I'll use `your` instead of `our` 😄